File: 125.cs
public class Solution {
    public bool IsPalindrome(string s) {
        List<char> list = new List<char>();
        s = s.ToLower();
        for(int i = 0; i<s.Length; i++){
            char c = s[i];
            if((c>='a' && c<='z') || (c>='0' && c<='9'))list.Add(c);
        }

        for(int i = 0; i<list.Count/2;i++){
            if(list[i]!=list[list.Count-i-1]) return false;
        } 
        return true;
    }
}

File: 125.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        left = 0
        right = len(s)-1
        while left < right:
            cleft = s[left]
            cright = s[right]
            if cleft.isalnum() == False:
                left+=1
                continue
            elif cright.isalnum() == False:
                right-=1
                continue

            cleft = cleft.lower()
            cright = cright.lower()
            
            if cleft != cright:
                return False
            
            left+=1
            right-=1
        
        return True
            

        
        